Explorer/Assets/DCL/AvatarRendering/Wearables/Components/Intentions/GetWearableByParamIntention.cs

```diff
@@ -19,12 +19,14 @@ public struct GetWearableByParamIntention : IEquatable<GetWearableByParamIntenti
 
         //Used for pooling
         public List<IWearable> Results;
+        public int TotalAmount;
```

Comment:
This field looks redundant. You only need it in `WearablesResponse`. Please check the flow

Reply:
It is the total amount of the whole backpack, not just of the results, I don't know where else I should put it